Makefile

```
@@ -7,7 +7,9 @@ SHELL = /usr/bin/env bash
 
 # Docker image info
 IMAGE_PREFIX ?= grafana
-IMAGE_TAG ?= $(shell ./tools/image-tag)
+ifeq ($(IMAGE_TAG),)
+IMAGE_TAG := $(shell ./tools/image-tag)
+endif
```

I was wrong, we don't really need this, sorry 😞. Here's an easy undo button.

```suggestion
IMAGE_TAG ?= $(shell ./tools/image-tag)
```